data/wanderers.txt

```diff
@@ -717,6 +737,9 @@ event "wanderers: tempest mass production"
 		remove variant
 			"Autumn Leaf"
 			"Summer Leaf"
+	fleet "Wanderer Miners"
+		add variant
+			"Tempest"
```

Comment:
This seems pretty overkill considering all heavier warships would be focused on defensive action and the war effort.